service/src/main/java/org/databiosphere/workspacedataservice/InstanceInitializerBean.java

```
@@ -163,12 +176,11 @@ private boolean initCloneMode() {
             // backup succeeded; now attempt to restore
             restoreFromRemoteBackup(backupFileName, cloneStatus.getJobId());
 
-            // after the restore attempt, check the current clone status one more time
-            // and return the result
+            // after the restore attempt, check the current clone status one more time,
+            // unlock, and return the result
             LOGGER.info("Re-checking clone job status after restore request");
             var finalCloneStatus = currentCloneStatus(trackingId);
             return finalCloneStatus.getStatus().equals(JobStatus.SUCCEEDED);
-
         }
         catch (Exception e) {
             LOGGER.error("An error occurred during clone mode. Error: {}", e.toString());
```

Looks like you didn't touch this block, but I'm wondering what you think of failing fast here instead of catching & swallowing the error, per [comment](https://github.com/DataBiosphere/terra-workspace-data-service/pull/341/files#r1331662613)

---

we don't necessarily want to fail fast in cloning, otherwise we wouldn't end up with any default schema. but i will look into fast fails elsewhere